runtime/doc/api.txt

```diff
@@ -512,27 +512,27 @@ nvim_parse_expression({expr}, {flags}, {highlight})
                                  [start_col, end_col)).
 
                 Return:~
-                    AST: top-level dictionary holds keys "error": Dictionary
-                    with error, present only if parser saw some error.
-                    Contains the following keys: "message": String, error
-                    message in printf format, translated. Must contain exactly
-                    one "%.*s". "arg": String, error message argument. "len":
-                    Amount of bytes successfully parsed. With flags equal to
-                    "" that should be equal to the length of expr string.
-                    @note: “Sucessfully parsed” here means “participated in
-                    AST creation”, not “till the first error”. "ast": AST,
-                    either nil or a dictionary with these keys: "type": node
-                    type, one of the value names from ExprASTNodeType
-                    stringified without "kExprNode" prefix. "start": a pair
-                    [line, column] describing where node is “started” where
-                    "line" is always 0 (will not be 0 if you will be using
-                    nvim_parse_viml() on e.g. ":let", but that is not present
-                    yet). Both elements are Integers. "len": “length” of the
-                    node. This and "start" are there for debugging purposes
-                    primary (debugging parser and providing debug
-                    information). "children": a list of nodes described in
-                    top/"ast". There always is zero, one or two children, key
-                    will not be present if node has no children. Maximum
```

Comment:
Looks horrible here while being fine in online docs.

Reply:
@ZyX-I Yes, need to enhance the doc-generation script somehow, or use doxygen "HTML". Not sure if doxygen discards the line break info.